addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -751,9 +751,34 @@ blockquote {
     }
```

Comment:
Probably mention it in https://www.odoo.com/odoo/all-tasks/4174638 ? But yes, it's probably the same bug ? You confirmed it is there without my branch too, right? :+1:

Reply:
(found out about `background-origin` indeed, that we should set to `border-box`, probably even in stable as it would only impact elements with both an image and a border (and with a small border, it would not be noticeable)). I added it in the task I created.